src/components/Button/Button.react.js

```diff
@@ -59,19 +61,8 @@ const Button = ({
       btn: true,
       [`btn-${size}`]: !!size,
       [`btn-block`]: block,
-      [`btn-primary`]: primary && !outline,
-      [`btn-secondary`]: secondary && !outline,
-      [`btn-success`]: success && !outline,
-      [`btn-info`]: info && !outline,
-      [`btn-warning`]: warning && !outline,
-      [`btn-danger`]: danger && !outline,
+      [`btn-${type}`]: type && !color,
```

Comment:
@jonthomp This line makes the type and color mutually exclusive - if we set the type and set the color then it will ignore the type.  Is this the desired functionality?  If so, feel free to approve.  If not then we should change this line.  Tagging you because you wrote the component so I'm unsure the desired functionality.

Reply:
I see your point. From the tabler docs the type modifiers (primary, secondary, etc) fall under the color section. So how about [these changes](https://github.com/m8r1x/tabler-react/commit/aca9ea50e594c70adf5f8c04cc7f89570be43a3d)